src/terms/nonlocal.jl

```diff
@@ -282,6 +381,7 @@ function PDPψk(basis, positions, psp_groups, kpt, kpt_minus_q, ψk)
     D = build_projection_coefficients(basis, psp_groups)
     P = build_projection_vectors(basis, kpt, psp_groups, positions)
     P_minus_q = build_projection_vectors(basis, kpt_minus_q, psp_groups, positions)
+    # TODO: probably needs an extra parenthesis to first compute P'ψ
```

Comment:
So... I noticed that Julia has custom `*` overloads for matrix multiplication with more than 2 operands to select the chain of operations that will minimize the total cost. Presumably it will almost always compute `P_minus_q' * ψk` first. But if it doesn't we are in trouble, so this should probably be changed to `(P_minus_q' * ψk)`.